react-native-fbsdk.podspec

```diff
@@ -14,17 +14,17 @@ Pod::Spec.new do |s|
   s.dependency      'React'
 
   s.subspec 'Core' do |ss|
-    ss.dependency     'FBSDKCoreKit', '~> 7.0'
+    ss.dependency     'FBSDKCoreKit', '~> 8.0'
```

Comment:
@janicduplessis these can go to 8.1 now, but I believe if you don't also update `example/ios/Podfile.lock` (or delete it) then CI will fail? It did on #811 but otherwise this is obviously a more interesting PR as it exposes functionality, not just bumping versions

Reply:
Updated to 8.1